src/DataWrangling/ECCO/ECCO.jl

```diff
@@ -200,7 +200,7 @@ function ECCO_field(metadata::ECCOMetadata;
     # ECCO4 data is on a -180, 180 longitude grid as opposed to ECCO2 data that
     # is on a 0, 360 longitude grid. To make the data consistent, we shift ECCO4
     # data by 180 degrees in longitude
-    if metadata.version isa ECCO4Monthly 
+    if metadata.version isa ECCO4Monthly || metadata.version isa ECCO4DarwinMonthly
```

Comment:
Will be slightly cleaner to use abstract type or type union to express this. This pattern does not generalize well; this makes it more work to change how we express `version` for example.

Reply:
as in `if metadata.version isa Union{ECCO4Monthly, ECCO4DarwinMonthly}`?